src/data/featuredContentData.ts

```diff
@@ -0,0 +1,26 @@
+const featuredContentData = [
```

Comment:
You can assign this the created type `const featuredContentData: FeaturedContentData[] = [ ...`.